src/pages/FilesPage/components/FolderItem/FolderItem.styles.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import FolderIcon from '@mui/icons-material/Folder';
+import Card from '@mui/material/Card';
+import Checkbox from '@mui/material/Checkbox';
+import Box from '@mui/material/Box';
+
+export const FolderItemIcon = styled(FolderIcon)`
+	font-size: 100px;
+	color: #6e52b3;
```

Comment:
We use this color at least two times for now. We can set colors for our @mui-ui theme and we will just write "primary", "secondary" and so on. We also have to do the same thing for font-sizes.